sdk/appcontainers/arm-appcontainers/package.json

```diff
@@ -25,7 +25,7 @@
   ],
   "license": "MIT",
   "main": "./dist/commonjs/index.js",
-  "module": "./dist-esm/src/index.js",
+  "module": "./dist/esm/index.js",
```

Comment:
@jeremymeng i am curios how we found this? not sure this is all impacted?

@kazrael2119 could you help check if the codegen is correct? do you know why?